docs/demos/TrafficManagerProfile/session-migrate-to-another-cluster/README.md

```diff
@@ -0,0 +1,190 @@
+            value:
+              { "networking.fleet.azure.com/weight": "50" }
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  cluster-name: member-5
+        jsonPatchOverrides:
+          - op: add
+            path: /metadata/annotations
+            value:
+              { "networking.fleet.azure.com/weight": "0" }
+```
+
+Apply the configuration:
+```sh
+kubectl apply -f eastus2euap/ro-nginx-service-export-eastus2euap.yaml
+```
+---
+
+## Step 2: Add label "env=prod" to `member-5`
+
+To ensure that `member-5` is recognized as a production environment, we need to add the label `env=prod` to it. This will allow the `clusterResourcePlacement` to propagate the resources correctly.
+
+```sh
+kubectl label cluster aks-member-5 env=prod
+```
+
+To check the status of the `clusterResourcePlacement`, we will run the following command. It's essential to ensure the resources are applied and available on the cluster before we shift the traffic.
+```sh
+kubectl get crp crp-team-a -o yaml
+```
+
+---
+
+## Step 3: Adjust Traffic Weight Using `resourceOverride`
+- Gradually shift requests from `member-1` to `member-5` by modifying the traffic weights in `serviceExport`.
+- Monitor traffic distribution by using `kubectl get tmb nginx-backend-eastus2euap -n team-a-nginx -o yaml` to ensure a smooth transition.
+
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ResourceOverride
+metadata:
+  name: ro-nginx-service-export-eastus2euap
+  namespace: team-a-nginx
+spec:
+  placement:
+    name: crp-team-a
+  resourceSelectors:
+    -  group: networking.fleet.azure.com
+       kind: ServiceExport
+       version: v1alpha1
+       name: nginx-service-eastus2euap
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  cluster-name: member-1
+        jsonPatchOverrides:
+          - op: add
+            path: /metadata/annotations
+            value:
+              { "networking.fleet.azure.com/weight": "0" }
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  cluster-name: member-3
+        jsonPatchOverrides:
+          - op: add
+            path: /metadata/annotations
+            value:
+              { "networking.fleet.azure.com/weight": "50" }
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  cluster-name: member-5
+        jsonPatchOverrides:
+          - op: add
+            path: /metadata/annotations
+            value:
+              { "networking.fleet.azure.com/weight": "50" }
+```
+---
+
+## Step 4: Clean Up Resources Using the `eviction` API
+- Remove the env label from the `member-1` so that the cluster won't be picked by the `clusterResourcePlacement` again.

```

Comment:
Just a random thought: rather than removing the `env` label would adding a taint be more appropriate (has a way smaller impact range)? No blocker, if it's just for demoing.